website/client/src/components/shops/balanceInfo.vue

```diff
@@ -67,33 +71,40 @@ export default {
   computed: {
     currencies () {
       const currencies = [];
-      currencies.push({
-        type: 'hourglasses',
-        icon: this.icons.hourglasses,
-        value: this.userHourglasses,
-      });
+      if (this.neededCurrencyOnly === true) {
```

Comment:
You don't need the equivalency operator, `if (this.neededCurrencyOnly) {` is sufficient

Reply:
Of course, derp!